packages/api/models/rules.js

```diff
@@ -174,7 +174,13 @@ class Rule extends Manager {
           item.rule.arn = mappingEnabled.UUID;
           return item;
         }
```

Comment:
It's not related to the PR, but `listData.EventSourceMappings.find` is called twice here.  Can we just check if `mappingExists` is enabled?

Reply:
I'll address in PR to this PR.

Reply:
@abarciauskas-bgse @jennyhliu for your consideration.  https://github.com/nasa/cumulus/pull/471